File: Python/Warmup-1/icy_hot.py

# this function return true if one of the two given temperatures is less than 0 and the other is greater than 100
def icy_hot(temp1, temp2):
	b = False

	if (temp1 < 0 and temp2 > 0) or (temp1 > 0 and temp2 < 0):
		b = True

	return b

print(icy_hot(120, -1))
print(icy_hot(-1, 120))
print(icy_hot(2, 120))

File: JavaScript/Warmup-1/icyHot.js

// given two temperatures, return true if one is less than 0 and the other is greater than 100
const icyHot = (temp1, temp2) => {
    return ((temp1 < 0 && temp2 > 100) || (temp1 > 100 && temp2 < 0)) ? true : false;
};

console.log(icyHot(120, -1));
console.log(icyHot(-1, 120));
console.log(icyHot(2, 120));
